docs/source/developer_guide/git_workflow.rst

```diff
@@ -10,14 +10,14 @@ GitHub, clone, and develop on a new branch.
 
 The workflow consists of two main parts:
 
-* **Creating a fork**: This sets up your own copy of the ``sktime`` repository on GitHub.
+* **Creating a fork and cloning the repository**: This sets up your own copy of the ``sktime`` repository on GitHub.
   This needs to be done only once, when you start contributing to ``sktime``.
 * **Developing a feature**: This is the process of developing a new feature, e.g.,
   a bugfix or new estimator.
   This is done every time you want to contribute a new feature.
 
 
-Creating a fork - Initial one time setup
+Creating a fork and cloning the repository
```

Comment:
Thoughts: since you specifically want to distinguish fork and original repository, will it make sense to also say "cloning the forked repository and not original sktime/sktime repository"? Or, a shortened form of that meaning?

Reply:
very valid point! I do want to avoid having such a long section name, so I will opt to clarify that meaning in the paragraph below.. will that work?